pkgs/tools/text/gnupatch/Abort_when_cleaning_up_fails.patch

```diff
@@ -0,0 +1,51 @@
+From b7b028a77bd855f6f56b17c8837fc1cca77b469d Mon Sep 17 00:00:00 2001
+From: Andreas Gruenbacher <agruen@gnu.org>
+Date: Fri, 28 Jun 2019 00:30:25 +0200
```

Comment:
Yes, no releases for 7 years, Debian applies this patch as well.

Reply:
I would be in favour of bumping to a Git revision given the number of fixes since then.

I think we should split out the two commits you mentioned into separate PRs so we can discuss things like that.

Reply:
Yes, absolutely agreed that a git version is the right call vs. vendoring the accumulated patches.

Reply:
https://git.savannah.gnu.org/cgit/patch.git/commit/?id=dce4683cbbe107a95f1f0d45fabc304acfb5d71a looks like the only potentially‐breaking interface change at a glance, but we can probably handle it. We have so many patches for `patch` right now and five of them are CVEs, we should just bump and nag upstream to cut a release.

(But yeah let’s talk about this on another PR!)